test/d-b-n_smoke/arrays.html

```diff
@@ -212,7 +243,7 @@
                         ppclient.addEventListener('patch-applied', patchAppliedEventCallback);
                         // change model
                         // puppetChange(puppet, [operationObject('add', '/people_properties/-', '1')]);
-                        puppetChange(puppet, [operationObject('add', '/people_properties/0', '1')]);
+                        puppetChange(puppet, [operationObject('add', '/people_properties/0', 'property')]);
```

Comment:
I think it's my oldest change in this changeset, I don't even remember why I did it :) I can probably change it to 'item' just fine.